doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -299,6 +299,19 @@ This section provides detailed lists of changes by :ref:`library <libraries>`.
 
   * :ref:`nrf_security` library, relocated from the sdk-nrfxlib repository to the :file:`subsys/nrf_security` directory.
 
+* :ref:`lib_hw_unique_key` library:
```

Comment:
Move both lists to Other libraries section.

Reply:
Done